q2_cutadapt/plugin_setup.py

```diff
@@ -212,7 +212,7 @@
         'discard_untrimmed': 'Discard reads in which no adapter was found.',
     },
     output_descriptions={
-        'trimmed_sequences': 'The resulting trimmed sequences.',
+        'trimmed_sequences': 'The resulting trimmed sequences.'
```

Comment:
Thinking about `git blame` --- does this change _need_ to be here? Looks like the comma was deleted (I suspect accidentally).

Reply:
Yeah, I don't even remember deleting that, easy to fix.